src/me/actions/creators/index.ts

```diff
@@ -1,11 +1,31 @@
 import {MeState} from 'src/me/reducers'
+import {Me} from 'src/client/unityRoutes'
+import {RemoteDataState} from 'src/types'
 
 export const SET_ME = 'SET_ME'
+export const SET_QUARTZ_ME = 'SET_QUARTZ_ME'
+export const SET_QUARTZ_ME_STATUS = 'SET_QUARTZ_ME_STATUS'
 
-export type Actions = ReturnType<typeof setMe>
+export type Actions =
+  | ReturnType<typeof setMe>
+  | ReturnType<typeof setQuartzMe>
+  | ReturnType<typeof setQuartzMeStatus>
 
 export const setMe = (me: MeState) =>
   ({
     type: SET_ME,
     me,
   } as const)
+
+export const setQuartzMe = (me: Me, status: RemoteDataState) =>
+  ({
+    type: SET_QUARTZ_ME,
+    quartzMe: me,
+    status,
+  } as const)
+
+export const setQuartzMeStatus = (status: RemoteDataState) =>
+  ({
+    type: SET_QUARTZ_ME_STATUS,
+    status,
+  } as const)
```

Comment:
I hear ya. I think the the latter is a consequence of having such a widespread network of actions that have caused accidental collisions in the past between dispatched actions with the same name. Since there isn't a centralized location of all actions, we have had situations where actions have mirrored the events that they are supposed to enact (e.g. UPDATE_TASK_NAME or whatever) but found out that there were unintended consequences due to an already existing action by the same name (that was a made up example of a real thing that happened early on in my time here where I named an action precisely based on what it was supposed to do and ended up encountering some weird redux issues where another action by the same name was triggered and the reducer state got out of wack). It's definitely worth considering a different model than the current codebase conventions to help with maintainability and for intended purposes rather than forcing it like we are here.